qa/suites/rados/cephadm-smoke/distro/centos_7.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/centos_7.yaml
```

Comment:
CentOS/RHEL 7 are not supported on Octopus, right? How would cephadm be expected to work on those?

Reply:
cephadm works on anything with python3 and podman, including older distros, since we're using the container image and not packages.

Reply:
(this is why we're adding a separate cephadm-smoke collection... the cephadm/ tests install ceph packages in order to run the client workloads, but cephadm-smoke does not.)